nix-plugins/tests/flox-nixpkgs.cc

```diff
@@ -68,17 +72,19 @@ test_lockedFromUrl( nix::ref<nix::EvalState> & state )
 {
   WrappedNixpkgsInputScheme inputScheme;
   auto                      url = "flox-nixpkgs:v0/flox/" + nixpkgsRev;
-  auto input = inputScheme.inputFromURL( nix::parseURL( url ), true );
+  auto                 input = inputScheme.inputFromURL( state->fetchSettings,
+                                         nix::parseURL( url ),
+                                         true );
   nix::fetchers::Attrs attrs
-    = inputScheme.fetch( state->store, *input ).second.toAttrs();
+    = inputScheme.getAccessor( state->store, *input ).second.toAttrs();
   auto owner      = nix::fetchers::getStrAttr( attrs, "owner" );
   auto flake_type = nix::fetchers::getStrAttr( attrs, "type" );
   auto rev        = nix::fetchers::getStrAttr( attrs, "rev" );
-  auto version    = nix::fetchers::getStrAttr( attrs, "version" );
+  auto version    = nix::fetchers::getIntAttr( attrs, "version" );
   EXPECT_EQ( owner, "flox" );
   EXPECT_EQ( flake_type, "flox-nixpkgs" );
   EXPECT_EQ( rev, nixpkgsRev );
-  EXPECT_EQ( version, "0" );
+  EXPECT_EQ( version, 0 );
```

Comment:
This generated a warning: https://github.com/flox/flox/actions/runs/12809336902/job/35779754519#step:7:66
```
../tests/flox-nixpkgs.cc:87:3: warning: comparison of integers of different signs: 'uint64_t' (aka 'unsigned long long') and 'int' [-Wsign-compare]
  EXPECT_EQ( version, 0 );
  ^~~~~~~~~~~~~~~~~~~~~~~
../tests/test.hh:136:15: note: expanded from macro 'EXPECT_EQ'
    if ( valA != valB )                                             \
         ~~~~ ^  ~~~~
1 warning generated.
```